NitroxClient/Communication/Packets/Processors/ItemContainerAddProcessor.cs

```diff
@@ -27,6 +27,12 @@ public override void Process(ItemContainerAdd packet)
             NitroxEntity.SetNewId(item, itemData.ItemId);
             
             itemContainer.AddItem(item, itemData.ContainerId);
+
+            // special Planting helper
+            if( itemData is PlantableItemData plantableData)
+            {
+                item.FixPlantGrowth(plantableData);
+            }
```

Comment:
Would prefer a ItemContainerAdd post processor.  Similar to pattern used for other things (BasePieceMetadata, DroppedItems, EntityMetadata).